images/win/scripts/Installers/Install-NodeLts.ps1

```diff
@@ -11,7 +11,16 @@ New-Item -Path $PrefixPath -Force -ItemType Directory
 New-Item -Path $CachePath -Force -ItemType Directory
 
 $defaultVersion = (Get-ToolsetContent).node.default
-Choco-Install -PackageName nodejs -ArgumentList "--version=$defaultVersion"
+$ODataQuery = "`$filter=(Id eq `'nodejs`') and (IsPrerelease eq false) and (Version ge `'$defaultVersion`')"
+$Url = "https://community.chocolatey.org/api/v2/Packages()?$ODataQuery"
+$ChocoVersionsOutput = (Invoke-RestMethod -Uri $Url).properties.Version
+$highestVersion = [Version[]]($ChocoVersionsOutput | Where-Object {$_ -Like "$defaultVersion.*"}) `
+    | Sort-Object -Descending `
+    | Select-Object -First 1
+# For case when $defaultVersion is the full version number
+if ( !$highestVersion ) { $highestVersion = $defaultVersion }
```

Comment:
Why do we need this condition if we have `Version ge '$defaultVersion'` filter?

Reply:
Correct, but in case $defaultVersion is the full version like "14.18.1" it will be filtered out by "| Where-Object { $_ -Like "$defaultVersion.*" }" so $highestVersion will be null here. Without "." it works incorrectly for versions like "14.1"

Reply:
We can add this check to `Where-Object { $_ -Like "$defaultVersion.*" } `